From 1136d2261d7f6a9a153709d0e2139a45d510a24d Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Tue, 5 Jun 2018 18:05:39 +0800
Subject: [PATCH 3911/5242] MLK-18483-01 soc: imx8: sc: types: add ipg stop
 misc controls for CONN ENET

commit  b3b0a78f749c9a3fcaafa7dd26e68b7273a68239 from
https://source.codeaurora.org/external/imx/linux-imx.git

- Sync with scu firmware commit 576011819ce3 (SCF-81: Added API to
control MIPI CSI calibration.) and commit 095a0d7dbc0b (SCF-85: Add
direct control of ENET IPG stop control)
- Add ipg stop misc controls for CONN ENET.

Reviewed-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/soc/imx8/sc/types.h |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/include/soc/imx8/sc/types.h b/include/soc/imx8/sc/types.h
index 3309e03..d9b57bf 100644
--- a/include/soc/imx8/sc/types.h
+++ b/include/soc/imx8/sc/types.h
@@ -763,6 +763,15 @@
 	SC_C_RST0 = 43,
 	SC_C_RST1 = 44,
 	SC_C_SEL0 = 45,
+	SC_C_CALIB0 = 46,
+	SC_C_CALIB1 = 47,
+	SC_C_CALIB2 = 48,
+	SC_C_IPG_DEBUG = 49,
+	SC_C_IPG_DOZE = 50,
+	SC_C_IPG_WAIT = 51,
+	SC_C_IPG_STOP = 52,
+	SC_C_IPG_STOP_MODE = 53,
+	SC_C_IPG_STOP_ACK = 54,
 	SC_C_LAST
 } sc_ctrl_t;
 
-- 
1.7.9.5

